packages/components/src/theme/components/checkbox.modifier.ts

```diff
@@ -1,7 +1,7 @@
 import { Components, Theme } from '@mui/material';
 import { CHARCOAL, connected, darkSlateGray } from '../colors';
 
-export const MuiCheckbox: Components<Theme>['MuiCheckbox'] = {
+export const SistentCheckbox: Components<Theme>['MuiCheckbox'] = {
```

Comment:
@sudhanshutech  using sistent as prefix is redundant ( as they will be scoped under sistent ) , I would suggest use `Base-<Component>` for unstyled / customizable component and `<Functionality-<Component>` for customized and ready to use ones  , @nebula-aac  whats your opinion ?

Reply:
@nebula-aac  @sudhanshutech    we dont need to have a complex naming convention , lets agree of something simple and stay consistent